start-testnet-client.sh

```diff
@@ -16,7 +16,7 @@ pushd "$(dirname "$0")"
 
 echo "Pulling down TestNet consensus validator signature material"
 
-SIGSTRUCT_URI=$(curl -s https://enclave-distribution.test.mobilecoin.com/production.json | grep sigstruct | awk '{print $2}' | tr -d \")
```

Comment:
Would maybe amend to:

```
SIGSTRUCT_URI=$(curl -s https://enclave-distribution.test.mobilecoin.com/production.json | jq -r '.consensus | .sigstruct')
```

And make sure to install jq earlier in the script

Reply:
Thanks! Done in 58b6889